docs/plotting.md

```diff
@@ -1,63 +1,79 @@
 # Plotting
+
 This page explains how to plot prices, indicator, profits.
 
 ## Installation
 
 Plotting scripts use Plotly library. Install/upgrade it with:
 
-```
-pip install --upgrade plotly
+``` bash
+pip install -U -r requirements-plot.txt
 ```
 
 At least version 2.3.0 is required.
```

Comment:
Is not this sentence redundant now and should be removed if requirements-plot.txt is now used? version 2.3.0 of what package, if reference to plotly is removed?